app/utils/validators/forms/FieldValidator.scala

```diff
@@ -136,4 +136,9 @@ object FieldValidator {
 
   private val emailPattern = Pattern.compile("""^\S+@\S+$""")
   val isValidEmail: String => Boolean = (name: String) => emailPattern.matcher(name).matches()
+
+  val REGEX = "^(GB|gb)[\\d]+$"
+  private val eoriPattern = Pattern.compile(REGEX)
```

Comment:
In general it's enough to inline this REGEX here

Reply:
Using the name eoriRegex as suggested above. Or shall it be changed to inline?

Reply:
Inline now..fixed